recipe/meta.yaml

```
@@ -45,32 +48,26 @@ test:
     - test -f ${PREFIX}/include/tango/tango.h                 # [unix]
     - if not exist %LIBRARY_BIN%\\tango.dll exit 1            # [win]
     - if not exist %LIBRARY_INC%\\tango\\tango.h exit 1       # [win]
-    - if not exist %LIBRARY_INC%\\pkgconfig\\tango.pc exit 1  # [win]
+    - if not exist %LIBRARY_LIB%\\pkgconfig\\tango.pc exit 1  # [win]
 
-outputs:
-  - name: cpptango
-    files:
-      - lib/libtango.so                     # [linux]
-      - lib/libtango.so.9                   # [linux]
-      - lib/libtango.so.{{ libversion }}    # [linux]
-      - lib/pkgconfig/tango.pc              # [linux]
-      - include/tango                       # [linux]
-      - Library/bin/tango.dll               # [win]
-      - Library/lib/tango.lib               # [win]
-      - Library/include/tango               # [win]
-      - Library/include/pkgconfig/tango.pc  # [win]
+outputs:                                                                 # [linux]
+  - name: cpptango                                                       # [linux]
+    files:                                                               # [linux]
+      - lib/libtango.so                                                  # [linux]
+      - lib/libtango.so.9                                                # [linux]
```

If I'm not mistaken, I think it should be the following instead:
```suggestion
      - lib/libtango.so.94                                                # [linux]
```

---

Thanks for catching this!